<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pointTypeDlg</class>
 <widget class="QDialog" name="pointTypeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>267</width>
    <height>381</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>267</width>
    <height>381</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>测点类型定义</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mainwindow.qrc">
    <normaloff>:/image/config.png</normaloff>:/image/config.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
     <item>
      <widget class="QLabel" name="typeLabel">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeCombo">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>类型描述</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>测点名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pointNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>测点属性</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="attrEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="modifyBtn">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>增加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delBtn">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loadBtn">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>加载默认值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>存储默认值</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>typeCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pointTypeDlg</receiver>
   <slot>typeComboChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifyBtn</sender>
   <signal>clicked()</signal>
   <receiver>pointTypeDlg</receiver>
   <slot>modifyPointType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delBtn</sender>
   <signal>clicked()</signal>
   <receiver>pointTypeDlg</receiver>
   <slot>delPointType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addBtn</sender>
   <signal>clicked()</signal>
   <receiver>pointTypeDlg</receiver>
   <slot>addPointType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadBtn</sender>
   <signal>clicked()</signal>
   <receiver>pointTypeDlg</receiver>
   <slot>loadDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>pointTypeDlg</receiver>
   <slot>saveDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>typeComboChanged(int)</slot>
  <slot>modifyPointType()</slot>
  <slot>addPointType()</slot>
  <slot>delPointType()</slot>
  <slot>loadDefault()</slot>
  <slot>saveDefault()</slot>
 </slots>
</ui>
